### PR TITLE
kernel/mqueue: Match types of variable between structures.

### PR DESCRIPTION
**Pull request by jeongarmy**
- Type of variables about message length, 'size_t'
   . maxmsgsize in mqueue_inode_s
   . msglen in mqueue_msg_s
   . msglen parameters of MQ functions.

- Type of variables about message count, 'size_t'
   . maxmsgs, nmsgs in mqueue_inode_s
   . mq_maxmsg, mq_msgsize in mq_attr.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [bda20afef141d239d450650941da1f5b681e8cc8] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [bda20afef141d239d450650941da1f5b681e8cc8] - Code Rule Check (C++) OK.

File: os/include/tinyara/mqueue.h [hunk 1]
```diff
@@ -85,15 +85,11 @@ struct mq_des;					/* forward reference */
 struct mqueue_inode_s {
 	FAR struct inode *inode;	/* Containing inode */
 	sq_queue_t msglist;			/* Prioritized message list */
-	int16_t maxmsgs;			/* Maximum number of messages in the queue */
-	int16_t nmsgs;				/* Number of message in the queue */
+	size_t maxmsgs;				/* Maximum number of messages in the queue */
```
Comment (jeongchanKim) [inline os/include/tinyara/mqueue.h]: Please remove unnecessary space.

File: os/include/tinyara/mqueue.h [hunk 2]
```diff
@@ -85,15 +85,11 @@ struct mq_des;					/* forward reference */
...
+	size_t nmsgs;				/* Number of message in the queue */
```
Comment (jeongchanKim) [inline os/include/tinyara/mqueue.h]: I think we don't need `size_t` size for `maxmsgs` and `nmsgs`.

**Comment by seinfra** [comment]
Target : [d2581e85e2ba3cc5667589070234b0c1eab02744] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d2581e85e2ba3cc5667589070234b0c1eab02744] - Code Rule Check OK.

**Pull request merged**